sdk/servicebus/service-bus/test/atomManagement.spec.ts

```diff
@@ -679,9 +522,6 @@ const alwaysBeExistingRule = "alwaysbeexistingrule";
     input: {
       lockDuration: "PT5M",
       maxDeliveryCount: 20,
-      // This should be a proper URL else the service returns an error
-      // To be investigated further as part of https://github.com/azure/azure-sdk-for-js/issues/6146
-      // forwardDeadLetteredMessagesTo: "",
```

Comment:
The test case title here is `all properties`. But the "forward" feature related properties are missing. Also missing are `status` and `userMetadata`.

I would suggest updating the `input` here to pass entity names for the "forward" feature properties. Then, we can remove the test case "pass in entity name for forwardTo and forwardDeadLetteredMessagesTo".

Reply:
Not all properties can be supplied at once. Setting forward related properties doesn't allow autoDeleteOnIdle to be set for instance.